docs/integrations/flask.md

```diff
@@ -16,7 +16,7 @@ app = Flask(__name__)
 
 app.add_url_rule(
     "/graphql",
-    view_func=GraphQLView.as_view("graphql_view", schema=schema),
+    view_func=GraphQLView.as_view("graphql_view", schema=Schema),
```

Comment:
I think the import is wrong, it should be `from api.schema import schema` because usually you export the generated sceham from `api.schema`

E.g

```
# api/schema.py

import strawberry

schema = strawberry.Schema(etc)

# in flask

from api.schema import schema

# then use schema
```

Reply:
I agree